posts/month-in-mathlib-nov-2021.md

```diff
@@ -0,0 +1,69 @@
+---
+author: 'Mathlib community'
+category: 'month-in-mathlib'
+date: 2021-12-01 06:58:04+00:00
+description: ''
+has_math: true
+link: ''
+slug: month-in-mathlib-nov-2021
+tags: ''
+title: This month in mathlib (Nov 2021)
+type: text
+---
+
+This post summarizes some of the activity that happened in mathlib in November.
+
+## Highlighted PRs
+
+* Differential equations.
+  - [PR #9791](https://github.com/leanprover-community/mathlib/pull/9791) Prove Picard-Lindelöf/Cauchy-Lipschitz Theorem
+
+* Combinatorics.
+  - [PR #10029](https://github.com/leanprover-community/mathlib/pull/10029) Hindman's theorem on finite sums
```

Comment:
The first one came a bit early and the last one a bit late, but we could mention the push of Kruskal-Katona towards mathlib:
```suggestion
  - [PR #10029](https://github.com/leanprover-community/mathlib/pull/10029) Hindman's theorem on finite sums
  - Set families are making their way to mathlib with [PR #9926](https://github.com/leanprover-community/mathlib/pull/9926), [PR #10223](https://github.com/leanprover-community/mathlib/pull/10223), [PR #10238](https://github.com/leanprover-community/mathlib/pull/10238) respectively defining antichains, shadow of a set family, UV compression. Those are ingredients for the LYM inequality, Sperner's theorem and the Kruskal-Katona theorem.
```
  -